src/components/InformationDialog/InformationDialog.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { spacing } from '@ui5/webcomponents-react-base';
+import { Icon } from '@ui5/webcomponents-react/lib/Icon';
+import { Button } from '@ui5/webcomponents-react/lib/Button';
+import { ButtonDesign } from '@ui5/webcomponents-react/lib/ButtonDesign';
+import { Dialog } from '@ui5/webcomponents-react/lib/Dialog';
+import { FlexBox } from '@ui5/webcomponents-react/lib/FlexBox';
+import { FlexBoxDirection } from '@ui5/webcomponents-react/lib/FlexBoxDirection';
+import { FlexBoxAlignItems } from '@ui5/webcomponents-react/lib/FlexBoxAlignItems';
+import { FlexBoxJustifyContent } from '@ui5/webcomponents-react/lib/FlexBoxJustifyContent';
+import { Text } from '@ui5/webcomponents-react/lib/Text';
+
+const KEYBOARD_KEYS = {
```

Comment:
I think this should be moved to the Constants file or another file since future components might use this.

Or we could use a library like `keycode-js` which is just a single js file exporting the constants for all keys: https://github.com/kabirbaidhya/keycode-js/blob/master/mod.ts

Reply:
Hmm, i have my doubts on that.
Every constants on a single file or every constant on its own context?